docs/Ordinal.md

```diff
@@ -1,13 +1,41 @@
 # Ordinal Support
 
-It is possible to use an ordinal as part of the collateral of a DLC.
-To do so, an [`OrdDescriptor`](../dlc-manager/src/contract/ord_descriptor.rs) must be used.
-This descriptor contains information about the ordinal being included in the DLC (location in the blockchain and the transaction that includes is), as well as information about the event upon which the DLC is based.
+This library now supports the locking of an ordinal as part of the collateral of a DLC.
+
+Notes:
+* An ordinal is inscribed into a single satoshi. However, that satoshi is often part of a larger UTXO. DLCs lock UTXOs, so please keep that context in mind.
+
+* In all cases, the gas fee to lock and later unlock the DLC is split between both parties.
+
+* For more information about DLCs, please see the main folder of this repo, the [`DLC-Link docs page`](https://docs.dlc.link), https://github.com/dlc-link, and the main https://github.com/discreetlogcontracts/dlcspecs specifications page.
+
+> Resizing UTXOs
+>
+> Please note, this package does not currently support the re-sizing of the UTXO containing the ordinal within the DLC. See [this section](#changing-postage) for more details
```

Comment:
Why use `>` here?

Reply:
Also since there is already a section about that under I feel this is redundant.

Reply:
OK, removed